libethashseal/Ethash.cpp

```diff
@@ -157,6 +157,8 @@ void Ethash::verifyTransaction(ImportRequirements::value _ir, TransactionBase co
 		}
 		else
 			_t.checkChainId(-4);
+		if (_bi.number() < chainParams().u256Param("metropolisForkBlock") && _t.hasZeroSignature())
```

Comment:
According to current `TransactionBase` constructor now `v` can't be 0 (will be rejected as invalid signature)